src/views/settings/Terminal/Monitor/component/MonitorCard.vue

```diff
@@ -99,7 +99,9 @@ export default {
         chen: 'Chen',
         kael: 'Kael',
         video_worker: 'Video-Worker',
-        panda: 'Panda'
+        panda: 'Panda',
+        nec: 'NEC',
+        facelive: 'Facelive'
       }
       return nameMapper[this.componentMetric.type]
     },
```

Comment:
There is a missing closing parenthesis on line 84 of file "index.js". It should be:

```
    return nameMapper[(this.componentMetric.type)]
```